### PR TITLE
Updated Scripts for apk-generator, dependencies for the project

### PR DESCRIPTION
**Pull request by mananwason**
@mariobehling @the-dagger Please review and merge

### CODE REVIEW

File: apk-generator/scripts/email.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo "Preparing to send the email"
+MPATH=$1
+EMAIL=$2
+mutt -s "Test mail" $EMAIL -a $MPATH/open-event-android/android/app/build/outputs/apk/app-googleplay-debug.apk < /var/www/mail_message.txt
+#mutt -s "test" manasbagula@gmail.com -a /root/Final/clone.sh < /root/mail_message.txt
```
Comment (the-dagger) [inline apk-generator/scripts/email.sh]: @mananwason There seems to be a strange email id associated with the comments in this bash file.
Please delete it.

Comment (mananwason) [inline apk-generator/scripts/email.sh]: Yes @the-dagger sure. I'll delete the comments

**Comment by mananwason** [comment]
@the-dagger Fixed.

**Comment by the-dagger** [comment]
Looks good, merging.

**Pull request merged**